plugins/pulp_rpm/plugins/controllers/errata.py

```diff
@@ -0,0 +1,44 @@
+import mongoengine
+
+from pulp_rpm.plugins.db.models import Errata, ErratumPkglist
+
+
+def create_or_update_pkglist(erratum, repo_id):
+    """
+    Create or update erratum pkglist and save it to DB.
+    Also make pkglist on Errata instance empty (in memory, not saved to DB yet).
+
+    :param erratum: An erratum unit being imported
+    :type erratum: pulp_rpm.plugins.db.models.Errata
+    :param repo_id: An id of repository into which the erratum unit is being imported
+    :type repo_id: str
+    """
+    pkglist_info = {'errata_id': erratum.errata_id,
```

Comment:
Nothing wrong here, although it seems a bit unexpected to make a dictionary only to then explode it as kwargs. Maybe you have some reason for it, but otherwise I think it would be natural to just pass these as named arguments when instantiating the class.